src/SearchResults.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import moment from "moment";
+import { useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+
+const SearchResults = props => {
+  const [color, setColor] = useState("transparent");
+  const [customerId, setCustomerId] = useState("");
+
+  function highlightRow(index) {
+    setColor(index === color ? "" : index);
+  }
+
+  const getCustomerId = id => {
+    setCustomerId(id);
+  };
+
+  return (
+    <div className="table-container">
+      <table className="table  results">
+        <thead>
+          <tr>
+            <th scope="col">Id</th>
+            <th scope="col">Title</th>
+            <th scope="col">First name</th>
+            <th scope="col">Surname</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room Id</th>
+            <th scope="col">Check in date</th>
+            <th scope="col">Check out date</th>
+            <th scope="col">Number of Night</th>
+          </tr>
+        </thead>
+        <tbody>
+          {props.results.map((item, index) => {
+            const startDate = moment(item.checkInDate);
+            const endDate = moment(item.checkOutDate);
+            let numberOfNights = endDate.diff(startDate, "days", true);
+
+            return (
+              <tr
+                className={color === index ? "red" : "transparent"}
```

Comment:
If you read the exercise carefully it says that the rows can be highlighted _independently_. This means that you can have more than 1 row highlighted at once.

How could you change your code to support that feature?

Reply:
Hi, 40thieves by your direction I have managed to make multiple row highlights and fixed the issue.